lib/internal/Magento/Framework/HTTP/Client/Curl.php

```diff
@@ -338,17 +338,18 @@ public function getStatus()
      * @param string $method
      * @param string $uri
      * @param array $params
+     * @param bool $json
      * @return void
      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
      * @SuppressWarnings(PHPMD.NPathComplexity)
      */
-    protected function makeRequest($method, $uri, $params = [])
+    protected function makeRequest($method, $uri, $params = [], $json = false)
```

Comment:
@okorshenko how should we do get/post requests? 

In our case we need to send post with json.